resolwe/observers/views.py

```diff
@@ -0,0 +1,55 @@
+"""Background task viewset."""
+
+from rest_framework import mixins, serializers, viewsets
+
+from resolwe.observers.mixins import ObservableMixin
+from resolwe.observers.models import BackgroundTask
+
+STATUSES = [status[0] for status in BackgroundTask.STATUS_CHOICES]
+
+
+class BackgroundTaskOptions(serializers.Serializer):
```

Comment:
Add this as a filter instead of a serializer.

Reply:
Done.